vrs/Dockerfile

```diff
@@ -0,0 +1,170 @@
+    && add-apt-repository -y https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/ \ 
+    && apt-get update && apt-get install -y adoptopenjdk-8-hotspot
+
+RUN mkdir tools 
+WORKDIR /tools
+
+# Install htslib
+# ENV SAMTOOLS_VERSION="1.11"
+# RUN wget https://github.com/samtools/htslib/releases/download/${SAMTOOLS_VERSION}/htslib-${SAMTOOLS_VERSION}.tar.bz2 \
+# 	&& tar xjf htslib-${SAMTOOLS_VERSION}.tar.bz2 \
+# 	&& rm htslib-${SAMTOOLS_VERSION}.tar.bz2 \
+# 	&& cd htslib-${SAMTOOLS_VERSION} \
+# 	&& ./configure \
+# 	&& make \
+# 	&& make install \
+# 	&& make clean
+
+
+# Install bcftools
+# RUN wget https://github.com/samtools/bcftools/releases/download/${SAMTOOLS_VERSION}/bcftools-${SAMTOOLS_VERSION}.tar.bz2 \
+# 	&& tar xjf bcftools-${SAMTOOLS_VERSION}.tar.bz2 \
+# 	&& rm bcftools-${SAMTOOLS_VERSION}.tar.bz2 \
+# 	&& cd bcftools-${SAMTOOLS_VERSION} \
+# 	&& ./configure \
+# 	&& make \
+# 	&& make install \
+# 	&& make clean
+
+# Install samtools
+# RUN wget https://github.com/samtools/samtools/releases/download/${SAMTOOLS_VERSION}/samtools-${SAMTOOLS_VERSION}.tar.bz2 \
+#	&& tar xjf samtools-${SAMTOOLS_VERSION}.tar.bz2 \
+# 	&& rm samtools-${SAMTOOLS_VERSION}.tar.bz2 \
+# 	&& cd samtools-${SAMTOOLS_VERSION} \
+# 	&& ./configure \
+# 	&& make \
+# 	&& make install \
+# 	&& make clean
+
+
+# install vcftools
+ENV VCFTOOLS_VERSION="0.1.16"
+RUN wget https://github.com/vcftools/vcftools/releases/download/v${VCFTOOLS_VERSION}/vcftools-${VCFTOOLS_VERSION}.tar.gz \
+    && tar xf vcftools-${VCFTOOLS_VERSION}.tar.gz \
+    && rm vcftools-${VCFTOOLS_VERSION}.tar.gz \
+	&& cd vcftools-${VCFTOOLS_VERSION} \
+	&& ./configure \
+	&& make \
+	&& make install \
+	&& make clean
+
+
+# install picard
+# ENV PICARD_VERSION="2.25.0"
+# RUN wget https://github.com/broadinstitute/picard/releases/download/${PICARD_VERSION}/picard.jar 
+
+# install GATK
+# ENV GATK_VERSION="4.2.0.0"
+# RUN wget https://github.com/broadinstitute/gatk/releases/download/${GATK_VERSION}/gatk-${GATK_VERSION}.zip \
+#    && unzip gatk-${GATK_VERSION}.zip \
+#    && rm gatk-${GATK_VERSION}.zip 
+
+
+# Install python packages
+RUN apt-get update \
+	&& apt-get dist-upgrade -y \
+	&& apt-get install -y --no-install-recommends\
+    libc6-dev \
+    libffi-dev \
+    libgdbm-dev \
+    liblapack-dev \
+    liblapack3 \ 
+    libncursesw5-dev \
+    libopenblas-base \
+    libopenblas-dev \
+    libsqlite3-dev \
+    libssl-dev \
+    openssl \
+    python3-smbus \
+    python3 python3-pip
+
+# Upgrade pip to latest version
+RUN python3 -m pip install --upgrade pip
+
+# Install hail and other python libraries
+# 02-13: at the end, include VRS annotation functionality
+ENV HAIL_VERSION="0.2.95"
+RUN python3 --version
+RUN python3 -m pip install \
+    wheel \
+    pypandoc \
+    hail==${HAIL_VERSION} \
+    scipy \
+    numpy \
+    pandas \
+    matplotlib \
+    seaborn \
+    ipython \
+    pybedtools \
+    dill \
+    psycopg2 \
+    ga4gh.vrs[extras] \
+    seqrepo
```

Comment:
i think you can just add "click" here to install it

Reply:
will do!